.github/workflows/acme-basic-test.yml

```diff
@@ -706,6 +695,68 @@ jobs:
           sed -n 's/^acmeStatus: *\(.*\)$/\1/p' output > actual
           diff expected actual
 
+
+      - name: Install caddy in client container
+        run: docker exec client dnf install -y caddy
+
+      - name: Configure ACME support in caddy
+        run: |
+          cat > Caddyfile << EOF
+          {
+            acme_ca https://pki.example.com:8443/acme/directory
+            acme_ca_root /etc/caddy/ca_signing.crt
+            key_type rsa2048
+          }
+          
+          client.example.com {
+            root * /usr/share/caddy
+            file_server
+          }
+          import Caddyfile.d/*.caddyfile
+          EOF
+
+          docker cp Caddyfile client:/etc/caddy
+          docker exec pki pki-server cert-export \
+              --cert-file $SHARED/ca_signing.crt \
+              ca_signing
+          docker exec client cp $SHARED/ca_signing.crt /etc/caddy/ca_signing.crt
+          
+      - name: Start caddy
+        run: |
+          docker exec client systemctl start caddy
+          # Wait caddy to start and get the certificate
+          sleep 40
+
+      - name: Check https is working
+        run: |
+          docker exec client curl -k /etc/caddy/ca_signing.crt https://client.example.com
```

Comment:
It might be nice if we could verify that the cert is created with ES256, but that's optional.

Reply:
The cert is normal RSA certificate. ES256 is a key used to create the communication between the client and the ACME server. It is stored in the account. After Caddy deploy you have 2 account and the second has the ES256 keu. [See this ](https://github.com/dogtagpki/pki/actions/runs/16168310131/job/45635728807?pr=5151#step:59:28). I'll add a check.